roles/sufia_dependencies/install/defaults/main.yml

```diff
@@ -11,3 +11,5 @@ hydra_pg_pass: my_hydra_pg_pass
 # needed for resque-pool init script
 capistrano_user: deploy
 fits_version: '0.8.4'
+# on debian, package version is now 9.4
+postgres_path: /etc/postgresql/9.3/main
```

Comment:
The comment in line 14 confuses me - I'd expect the path in line 15 to say 9.4 not 9.3